src/actions/views/root.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {NavigationTypes} from 'constants';
+import Routes from 'navigation/routes';
+
```

Comment:
I'm actually thinking of eventually moving all of these into view action creators and not having a separate path for navigation ones. With the navigation being a stack, each navigation action creator will be likely depend heavily on which view in. We just need a NAVIGATION_PUSH to get from SelectServer to Login, but to get from SelectTeam to Login, we need a NAVIGATION_POP.